### PR TITLE
Added tests

### PR DESCRIPTION
**Pull request by JDDV**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)
Added tests for platform-handler

### :boom: Does this PR introduce a breaking change?
No

### :thinking: Checklist before submitting

- [x] I made sure all projects build.
- [x] I updated pubspec.yaml with an appropriate new version according to the pub versioning philosophy.
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I followed the style guide lines ([code style guide](https://github.com/Baseflow/flutter-permission-handler/blob/develop/CONTRIBUTING.md)).
- [x] I updated the relevant documentation.
- [x] I rebased onto current `master`.

### CODE REVIEW

File: permission_handler_platform_interface/lib/permission_handler_platform_interface.dart [hunk 1]
```diff
@@ -1,6 +1,7 @@
 library permission_handler_platform_interface;
 
 import 'dart:async';
+import 'package:flutter/cupertino.dart';
```
Comment (mvanbeusekom) [inline permission_handler_platform_interface/lib/permission_handler_platform_interface.dart:4]: Do we need this import?

File: permission_handler_platform_interface/lib/src/permission_status.dart [hunk 2]
```diff
@@ -3,10 +3,10 @@ part of permission_handler_platform_interface;
 /// Defines the state of a [Permission].
 enum PermissionStatus {
   /// The user granted access to the requested feature.
-  granted,
+  denied,
 
   /// The user denied access to the requested feature.
-  denied,
+  granted,
```
Comment (mvanbeusekom) [inline permission_handler_platform_interface/lib/src/permission_status.dart:9]: The comments don't match the state anymore. These should be switched as well.

**Review by mvanbeusekom — CHANGES_REQUESTED** [review]
Looks good but I have two small remarks.

File: permission_handler_platform_interface/lib/permission_handler_platform_interface.dart (hunk 1 above)
Comment (JDDV) [reply permission_handler_platform_interface/lib/permission_handler_platform_interface.dart:4]: changed to meta/meta.dart import

**Review by mvanbeusekom — APPROVED** [review]
LGTM

**Pull request merged**